nvflare/lighter/utils.py

```diff
@@ -273,6 +273,12 @@ def update_storage_locations(
         outfile.write(json_object)
 
 
+def make_dirs(dirs):
+    for d in dirs:
+        if not os.path.exists(d):
+            os.makedirs(d)
```

Comment:
the same as:
       os.makedirs(d, exist_ok=True)

Reply:
Yes, got this from original code and just moved here. Could improve later.